src/impl.cpp

```diff
@@ -1 +1,22 @@
 #include "impl.h"
+#include <iostream>
+#include <istream>
+#include <vector>
+#include <iterator>
+#include <sstream>
+#include <algorithm>
+#include <unordered_set>
+
+std::vector<std::string> uniqueWord(const std::string string)
+{
+    std::string text = std::move(string);
```

Comment:
**now think what will happen.**
static_cast<const std::string&&>(string) -> const std::string&& -> /reference will be collapsed/ -> const std::string&

Which constructor will be called?
What the profit of the std::move here?